### DISCUSSION TITLE
Can't flush cache from dashboard or CLI

### DISCUSSION THREAD

**Discussion by ken-cdit** [rhubarbgroup/.github#56] [Support (OCP)]
I am using Object Cache Pro included with Pantheon plan.  Dashboard works, caching is working, license ok, drop-in ok.

Flushing cache with the dashboard button or with `wp cache flush` results in:

`Error: The object cache could not be flushed.`

**Comment by tillkruss** [comment]
We suggest asking the Pantheon team, they are closely familiar with the Object Cache Pro setup on their infrastructure.

My suggestion would be to enable logging and take a closer look at what's causing the error.

**Reply by ken-cdit** [reply]
When we came in this morning, the dashboard says drop-in invalid and nothing else is working.

I think we are just going to remove this plugin.

**Reply by tillkruss** [reply]
"drop-in invalid" usually mean you're using the wrong `wp-content/object-cache.php` drop-in, maybe from another object caching plugin like LSCache, or W3 Total Cache?
